flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
disappointing.

Reply:
flatpak/flatpak-builder-tools#383 blocked me.

Reply:
maybe you can help and contribute.

Reply:
I now spend all my spare time developing the build and distribution package process, and I don't have any spare time left😭, sometimes I even forget to eat😭😭.